apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -578,6 +581,8 @@ private boolean isBackdatedBene(RifRecordEvent<?> rifRecordEvent) {
    * @return {@code true} if the bene should be filtered/skipped
    */
   private boolean isBackdatedBene(Beneficiary bene) {
+    // no filtering of synthetic data
+
```

Comment:
This seems like it was moved out, so should this comment be removed/moved?

Reply:
removed.